techniques/flash/FLASH26.html

```diff
@@ -28,7 +28,7 @@ <h3>Playing descriptions when cue points are reached</h3>
     						provided below the video player that allows the user to turn
     						audio descriptions on or off. </p>
          
-         <pre xml:space="preserve">package {
+         <pre>package {
```

Comment:
all the naked `<pre> ... </pre>` blocks in this technique could do with inner `<code> ... </code>`